HelpAssistHer/help-assist-her
# No results view

## Description
This PR adds the "No Results" view, when there aren't any results for the location data that was entered.

Designs: https://projects.invisionapp.com/d/main#/console/15837992/328667331/preview

<img width="1435" alt="Screen Shot 2019-05-29 at 9 28 25 AM" src="https://user-images.githubusercontent.com/6378435/58574142-2d170f00-81f4-11e9-8360-102d0c3e0477.png">

## Test Plan
1. Go to http://localhost:4000/mini-app
1. Type in a location that won't have any results (I used "San Francisco")
1. Verify that you see the "No results" page
1. Verify that it looks good in the phone view as well
1. Verify that it matches the designs for both desktop and phone version
